features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/StreamingAPI/APICreateStreamingAPI.jsx

```diff
@@ -41,11 +42,34 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const APICreateStreamingAPI = (props) => {
-    const { history } = props;
+    const { history, intl } = props;
```

Comment:
Better to use `useIntl` here

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10269/commits/393f067f9a2fdb3cefa523854b54ced3f897161d